.github/workflows/build.yml

```diff
@@ -48,10 +48,18 @@ jobs:
           npm install @playwright/test@${{ matrix.playwright }}
 
       - name: Check types, run lint + tests
+        if: ${{ matrix.playwright == 'latest' }}
         run: |
           npm why playwright
           npm why @playwright/test
-          npm run validate
+          npm run test
+
+      - name: Check types, run lint + tests
```

Comment:
The old version of Playwright we still test against (1.12) does not have the `Locator` APIs. I'll add a comment here to explain that until we drop it.